Exercises/2-elements.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
 const removeElements = (array, ...items) => {
-  // Remove multiple items from array modifying original array
+  for (const i of items) {
```

Comment:
Naming `i` may confuse someone (may refer to `i:Number` like in regular for loops), better use `item` or `element` here.